packages/design/package.json

```diff
@@ -8,16 +8,22 @@
   "files": [
     "dist/*",
     "*.css",
+    "foundation.*",
     "icons/*"
   ],
   "scripts": {
-    "build": "npm run build:css && npm run build:css:icons && npm run build:colors && npm run build:cssTypes && npm run build:rollup",
+    "build": "npm run build:css && npm run build:css:icons && npm run build:colors && npm run build:cssTypes && npm run build:foundation && npm run build:rollup",
     "build:rollup": "rollup --config",
     "build:css": "postcss src/foundation.css --output foundation.css --use postcss-import",
     "build:css:icons": "postcss src/icons/*.css --dir icons/ --use postcss-import",
     "build:ts": "tsc --project .",
     "build:cssTypes": "tcm src/icons",
     "build:colors": "node buildColors.js",
+    "build:foundation": "node buildFoundation.js && node jobberStyle.js && npm run build:foundationTypes && npm run build:addTypeLinterExceptions && npm run build:addStyleLinterExceptions && npm run build:removeTempFiles",
+    "build:foundationTypes": "tsc --declaration --emitDeclarationOnly --allowJs foundation.js --outDir .",
+    "build:addTypeLinterExceptions": "(printf '/* tslint:disable */\n/* eslint-disable */\n/* This file is automatically generated and should not be edited. */\n'; cat foundation.d.ts) > foundation.d.ts.temp",
+    "build:addStyleLinterExceptions": "(printf '/* tslint:disable */\n/* eslint-disable */\n/* This file is automatically generated and should not be edited. */\n'; cat foundation.js) > foundation.js.temp",
+    "build:removeTempFiles": "rm -f src/foundation.js && mv foundation.d.ts.temp foundation.d.ts && mv foundation.js.temp foundation.js",
```

Comment:
I wonder if it would be nicer to make a `tmp` directory that we can just git ignore then we can skip this explicit cleanup step?

Reply:
The previous steps can't write to files in place so we write to tmp files and then make the tmp file the real ones (See the `mv` parts). So we would still need most of it. It would remove the `rm -f` part, I can do it if you think this would benefit from that but the `mv` steps are still required to happen